packages/openapi3/test/xml-models.test.ts

```diff
@@ -0,0 +1,705 @@
+import { deepStrictEqual } from "assert";
+import { describe, it } from "vitest";
+import { oapiForModel } from "./test-host.js";
+
+describe("@name", () => {
+  it("set the element value via @name", async () => {
+    const res = await oapiForModel(
+      "Book",
+      `
+        @name("xmlBook")
+        model Book {        
+          content: string;
+        };`
+    );
+
+    deepStrictEqual(res.schemas.Book, {
+      type: "object",
+      properties: {
+        content: { type: "string" },
+      },
+      required: ["content"],
+      xml: {
+        name: "xmlBook",
+      },
+    });
+  });
+
+  it("set the element value for array property via @name", async () => {
```

Comment:
not sure what this test is doing? how is the array property relevant to the fact that this is using `@name` on the model?